integration/kubernetes/k8s-parallel.bats

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+
+setup() {
+	export KUBECONFIG=/etc/kubernetes/admin.conf
+	if sudo -E kubectl get runtimeclass | grep -q kata; then
+		pod_config_dir="${BATS_TEST_DIRNAME}/runtimeclass_workloads"
+	else
+		pod_config_dir="${BATS_TEST_DIRNAME}/untrusted_workloads"
+	fi
+}
+
+@test "Parallel jobs" {
+	job_name="jobtest"
+	local -a names=( test1 test2 test3 )
+	# Create yaml files
+	for i in "${names[@]}"; do
+		cat ${pod_config_dir}/job-template.yaml | sed "s/\$ITEM/$i/" > ${pod_config_dir}/job-$i.yaml
```

Comment:
pipe is not needed

`sed "s/\$ITEM/$i/" ${pod_config_dir}/job-template.yaml > ${pod_config_dir}/job-$i.yaml`